MUI Avatar dark mode style issue

shubham-eaton opened the pull request:
<!-- If this pull request fixes an Issue, link it below. If not, you can remove the line below -->

Fixes # .
- Fixed MUI avatar dark mode issue #107 

<!-- Include screenshots if they will help illustrate the changes in this PR -->

#### Screenshots / Screen Recording (if applicable)

- 
<img width="1840" alt="Screenshot 2024-11-13 at 4 18 06 PM" src="https://github.com/user-attachments/assets/8fa89837-08ac-4b8b-9f26-54c8f53af79f">


<!-- Instruction for PR reviewers, if more complicated than a simple yarn start -->

#### To Test:

- cd demo/showcase
- yarn add @mui/codemod @mui/docs @mui/envinfo @mui/icons-material @mui/lab @mui/material @mui/private-theming @mui/styled-engine @mui/styled-engine-sc @mui/styles @mui/system @mui/types @mui/x-date-pickers
- yarn start
- Open RTLProvider.tsx
- import -> import {blueThemes} from '@brightlayer-ui/react-themes'
- Replace theme provider -> <ThemeProvider theme={blueThemes} defaultMode='light' >

JeffGreiner-eaton commented:
I tried testing this out on the showcase dev branch using "@brightlayer-ui/react-themes": "^8.0.0-alpha.0"
- Installed all packages listed above plus themes 8 alpha.0
- I updated the StylesOverrides MuiAvatar files from dist
- updated RTLProvider

![image](https://github.com/user-attachments/assets/c759321d-583a-469a-8639-ee3ce5b1a0b8)

![image](https://github.com/user-attachments/assets/3e1441b1-bd0c-4fb5-860f-8fda976909cf)

and I see issue
![image](https://github.com/user-attachments/assets/edc0a7d7-7fb8-4463-a083-fb9a0ce23a7a)

If I use your branch feature/mui-6 for showcase everything looks fine. Can we update the showcase to have all updated dependencies (installs from test instructions) and open that PR

shubham-eaton commented:
> I tried testing this out on the showcase dev branch using "@brightlayer-ui/react-themes": "^8.0.0-alpha.0"
> 
> * Installed all packages listed above plus themes 8 alpha.0
> * I updated the StylesOverrides MuiAvatar files from dist
> * updated RTLProvider
> 
> ![image](https://private-user-images.githubusercontent.com/119693939/385829039-c759321d-583a-469a-8639-ee3ce5b1a0b8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE1OTE0ODEsIm5iZiI6MTczMTU5MTE4MSwicGF0aCI6Ii8xMTk2OTM5MzkvMzg1ODI5MDM5LWM3NTkzMjFkLTU4M2EtNDY5YS04NjM5LWVlM2NlNWIxYTBiOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTE0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExNFQxMzMzMDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lNGIxNGFhZTQ0ODJmNmYyOTI4ZjgyYTZmYzU2OWNjOWUxODZhMzNhOGJkMGIwMTQ4YmQ0NjgwNzZhNDBhMzMwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.5mBJnlw6DLZjytj0xvtrFSIPnUOtt7snklkSFcCoZC8)
> 
> ![image](https://private-user-images.githubusercontent.com/119693939/385827710-3e1441b1-bd0c-4fb5-860f-8fda976909cf.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE1OTE0ODEsIm5iZiI6MTczMTU5MTE4MSwicGF0aCI6Ii8xMTk2OTM5MzkvMzg1ODI3NzEwLTNlMTQ0MWIxLWJkMGMtNGZiNS04NjBmLThmZGE5NzY5MDljZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTE0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExNFQxMzMzMDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01ODZhNjI3MmQ4ZmI4NDMxNWE2YWEzNjJjYjFmMjg1MzBiNjliOGJiMzFhZDg4NjBjNmIzYzgxZDU2M2YyODRlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.uAYxFflIEJ90KueZi1kBMH4Ufuo5z8e5HGsswdXkE-U)
> 
> and I see issue ![image](https://private-user-images.githubusercontent.com/119693939/385828264-edc0a7d7-7fb8-4463-a083-fb9a0ce23a7a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE1OTE0ODEsIm5iZiI6MTczMTU5MTE4MSwicGF0aCI6Ii8xMTk2OTM5MzkvMzg1ODI4MjY0LWVkYzBhN2Q3LTdmYjgtNDQ2My1hMDgzLWZiOWEwY2UyM2E3YS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTE0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExNFQxMzMzMDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hNTEwN2MxNmIwMjUwZmUyOGJmNGY0MjJiZmRlZGFjMjAzYzRmZWQ5ODA0MzJkYTA5ZTY5M2EyMzFjYjRlYTBjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.5VUGL-fvw1wEbWNNXIZhzg08QbIKyWR2enbK9ML-bRk)
> 
> If I use your branch feature/mui-6 for showcase everything looks fine. Can we update the showcase to have all updated dependencies (installs from test instructions) and open that PR feature/mui-6

Delete node modules and yarn lock then do run command "yarn cache clean"  then run yarn start. For me also it was not working until I cleaned cache of yarn. Also I am updating showcase with MUI changes and updating components to use MUI 6, will raise PR for it by tomorrow.

shubham-eaton commented:
> ok that worked. Can we get a PR open for this branch and updated? https://github.com/etn-ccis/blui-react-showcase-demo/tree/feature/mui-6

The upgrade of showcase is almost done, I am facing an issue while upgrading. Will raise a PR asap after its resolution.
